src/jit/lower.cpp

```diff
@@ -3605,6 +3605,7 @@ void Lowering::InsertPInvokeCallProlog(GenTreeCall* call)
     }
     else
     {
+#ifndef FEATURE_READYTORUN_COMPILER
```

Comment:
Why is this needed?

Reply:
The call to `embedMethodHandle` is not R2R compatible (There's an assert to make sure we don't call it). I need to look into this further.
But also, i'm not sure why we need it for pinvoke. It seems to work without it